src/app/pad.component.html

```diff
@@ -136,6 +136,19 @@
             <a class="invert" href="https://github.com/blindpad/blindpad/issues/new">feedback</a>
             <a class="invert" href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=VW2QSSP76L36N">donate</a>
             <a class="invert" routerLink="/about">about</a>
+
+
+            <button class="keymap-button" *ngIf="!!getPadKeymap()" (click)="onKeymapButtonClick()">
```

Comment:
@rmnoon Any idea why it is not displaying? `getPadKeymap` evaluates to null for some reason even if I set the default value to `'vim'`